vendor/k8s.io/kubernetes/pkg/volume/configmap/configmap.go

```diff
@@ -194,9 +194,6 @@ func (b *configMapVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 	if err := wrapped.SetUpAt(dir, fsGroup); err != nil {
 		return err
 	}
-	if err := volumeutil.MakeNestedMountpoints(b.volName, dir, b.pod); err != nil {
```

Comment:
Please fix this, you reverted changes.

Reply:
Hmm, maybe because we did the `glide get` and `update-deps.sh` a few weeks back - this might be outdated. Will do.